client/reader/site-stream/featured.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import page from 'page';
+import PostStore from 'lib/feed-post-store';
+import FeedPostStoreActions from 'lib/feed-post-store/actions';
+import stats from 'reader/stats';
+
+export default React.createClass( {
+	displayName: 'FeedFeatured',
+
+	getInitialState() {
+		return this.getStateFromStores();
+	},
+
+	getStateFromStores( store ) {
+		store = store || this.props.store;
+		return {
+			posts: store.get()
+		};
+	},
+
+	updateState( store ) {
+		this.setState( this.getStateFromStores( store ) );
+	},
+
+	componentDidMount() {
+		this.props.store.on( 'change', this.updateState );
+		PostStore.on( 'change', this.updateState );
+	},
+
```

Comment:
need a componentWillReceiveProps too to detect when the store changes and we need to regather state. This could happen when going from one stream that has features to another.